src/main/webapp/WEB-INF/tags/instructor/comments/visibilityOptions.tag

```diff
@@ -20,7 +29,7 @@
     <div id="visibility-options${commentIdx}" class="panel panel-default"
         style="display: none;">
         <div class="panel-heading">Visibility Options</div>
-        <table class="table text-center text-color-black">
+        <table class="table text-center" style="color: #000;">
```

Comment:
Why not use text-color-black, it's more descriptive than the RGB value of 00 00 00, plus using CSS > using inline styles, or is it more HTML changes needed when we use classes?